src/platform/packages/shared/kbn-esql-ast/src/antlr/esql_parser.g4

```diff
@@ -329,8 +329,10 @@ lookupCommand
     : DEV_LOOKUP tableName=indexPattern ON matchFields=qualifiedNamePatterns
     ;
 
-inlinestatsCommand
-    : DEV_INLINESTATS stats=aggFields (BY grouping=fields)?
+inlineStatsCommand
+    : DEV_INLINE INLINE_STATS stats=aggFields (BY grouping=fields)?
```

Comment:
@bartoval this is what we want to promote. The old one (inlinestats one word) is deprecated, we dont want to suggest it so we need to change our inlinestats to inline stats